check.go

```diff
@@ -289,25 +289,20 @@ func (c *CheckRunner) UpdateCheck(checkID types.CheckID, status, output string)
 		return
 	}
 
-	// Do nothing if update is idempotent
-	if check.Status == status && check.Output == output {
-		check.failureCounter = decrementCounter(check.failureCounter)
-		check.successCounter = decrementCounter(check.successCounter)
```

Comment:
These are the only 2 uses of decrementCounter(). Should probably nuke it as well.